data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,457 @@
+		outfit "Jump Drive"
+	on visit
+		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
+	on complete
+		payment 1000000
+		outfit "Jump Drive" -1
+		dialog `The Unfettered are more than content with paying <payment> credits in exchange for your precious cargo.`
+
+mission "Get Jump Drives(2)"
+	name `Collect 2 jump drives`
+	description `Go to the pirate world of <destination> and recover 2 jump drives from a vault for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	destination "Darkcloak"
+	to offer
+		random < 45
+		has "Unfettered: Prove your strength 2: done"
+	stopover
+		distance 12 18
+		government "Pirate"
+	on stopover
+		dialog `You manage to land and retrieve the jump drives without too much trouble. Do not rely on luck too long though, you had better depart quickly!`
+		outfit "Jump Drive" 2
+	on visit
+		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
+	on complete
+		payment 2000000
+		outfit "Jump Drive" -2
+		dialog `The Unfettered are more than content with paying <payment> credits in exchange for your precious cargo.`
+
+mission "Get Jump Drives(3)"
+	name `Collect 3 jump drives`
+	description `Go to the pirate world of <destination> and recover 3 jump drives from a vault for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	destination "Darkcloak"
+	to offer
+		random < 15
+		has "Unfettered: Prove your strength 2: done"
+	stopover "Deadman's Cove"
+	on stopover
+		dialog `You manage to land and retrieve the jump drives without too much trouble. Do not rely on luck too long though, you had better depart quickly!`
+		outfit "Jump Drive" 3
+	on visit
+		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
+	on complete
+		payment 3000000
+		outfit "Jump Drive" -3
+		dialog `The Unfettered are more than content with paying <payment> credits in exchange for your precious cargo.`
+
+
+
+mission "Bring True Hai"
+	name `Bring True Hai`
+	description `Land on <stopovers> and bring back <bunks> True Hai to where they belong, on <destination> for <payment>.`
+	job
+	repeat
+	passengers 5 8 .2
+	source
+		attributes "unfettered"
+	destination
+		attributes "unfettered"
+	to offer
+		random < 75
+		has "Unfettered: The Word Has Spread: done"
+	stopover
+		distance 3 6
+		government "Hai"
+		attributes "spaceport"
+	on stopover
+		dialog `You have picked up all <bunks> Hai, now all that is left is to bring them to <destination>.`
+	on complete
+		payment
+		payment 150000
+		dialog `You feel good that you managed to bring these Hai to their True home. The Unfettered hand you the agree payment of <payment>.`
```

Comment:
Generally speaking, the game isn't supposed to tell the player how they feel about something. 
```suggestion
		dialog `You feel good that you managed to bring these Hai to their True home. The Unfettered hand you the agreed payment of <payment>.`
```